### PR TITLE
[SCB-1497]support for skywalking.

### PR DESCRIPTION
**Pull request by surechen**
SCB-1497

Signed-off-by: surechen <chenshuo17@huawei.com>

### CODE REVIEW

File: docs/skywalking/skywalking.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      enable: true
+      serverUri: 127.0.0.1:11800
+```
+
+## SkyWawlking-Manager Init
+**In file proxy/bootstrap/bootstrap.go**
+```shell script
+import "github.com/apache/servicecomb-mesher/proxy/pkg/skywalking"
+```
+**In function Start()**
```
Comment (tianxiaoliang) [inline docs/skywalking/skywalking.md]: should teach how to use configuration file， skywalking
 is not enabled in chain by default

File: proxy/config/struct.go [hunk 2]
```diff
@@ -66,3 +67,19 @@ type Admin struct {
...
 }
+
+//Tracing has attributes for APM
+type Tracing struct {
+	Enable    bool   `yaml:"enable"`
+	ServerURI string `yaml:"serverUri"`
+}
+
+//Apm is for Application Performance Management
+type Apm struct {
```
Comment (tianxiaoliang) [inline proxy/config/struct.go]: all upper case APM

File: docs/skywalking/skywalking.md (hunk 1 above)
Comment (tianxiaoliang) [reply docs/skywalking/skywalking.md]: and do not teach so many code that you already wrote

Comment (surechen) [reply docs/skywalking/skywalking.md]: Modify according to code review comments

File: proxy/config/struct.go (hunk 2 above)
Comment (surechen) [reply proxy/config/struct.go]: Modify according to code review comments

**Pull request merged**